### PR TITLE
enhance the note about remove ptf capability

### PR DESCRIPTION
**Pull request by mcalmer**
# Description

Give a hint how to find out why a PTF cannot be removed.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3
- [ ] Manager-4.2

### CODE REVIEW

**Comment by vandabarata** [comment]
Thank you for the addition :grin:

**Review by vandabarata — APPROVED** [review]

File: modules/administration/pages/ptfs.adoc [hunk 1]
```diff
@@ -95,6 +95,7 @@ a simple package remove cannot be used. The ptf package is not selectable in the
 [NOTE]
 ====
 Removing PTFs requires a special version of libzypp/zypper to be installed on the client system.
+Check ``zypper --help`` if the command ``removeptf`` is supported.
```
Comment (keichwa) [inline modules/administration/pages/ptfs.adoc:98]:
```suggestion
Removing PTFs requires a special version of [literal]``libzypp`` and [literal]``zypper`` to be installed on the client system.
Check [command]``zypper --help`` whether [command]``removeptf`` is supported.
```
Evenif technically correct, I think there is no need to mention `libzypp` here.

File: .changelog [hunk 2]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- enhance the note about remove ptf capability
```
Comment (keichwa) [inline .changelog:7]:
```suggestion
- Enhanced the note about the remove PTF capability in Administration Guide
```

**Review by keichwa — APPROVED** [review]

**Review by 0rnela — APPROVED** [review]

**Comment by keichwa** [comment]
If you do not veto, @mcalmer , I'd like to merge and then backport this PR.  Of course, if wanted you can also do it yourself.

**Pull request merged**